src/helpers/find.case.by.id.js

```diff
@@ -1,8 +1,15 @@
 let findCaseById = function(options, then) {
-    let found = options.cases.find((item) => item.id === options.id );
```

Comment:
seriously? IE doesn't support this? :(

Reply:
you see my fun?

Reply:
I think you still might be able to use javascript map reduce, no?

Reply:
https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Array/find#Compatibilit%C3%A9_des_navigateurs

Reply:
en français!

Reply:
so much fun